src/views/Settings.vue

```diff
@@ -307,12 +359,18 @@ export default defineComponent({
         logger.error('Failed to update facility', err)
       }
     },
+    async updateFulfillmentStatus(event: any) {
+      if (event.detail.checked === this.fulfillmentStatus) return
```

Comment:
Add comment why we have added this condition.

Reply:
Updated sir